std/process.d

```diff
@@ -1358,6 +1476,15 @@ final class Pid
         return _processID;
     }
 
+    /**
+    This is $(D true) if process was spawned without $(LREF Config.detached), $(D false)
+    otherwise. You can't $(LREF wait) for or $(LREF kill) process you don't own.
+    */
+    @property bool owned() const @safe pure nothrow
```

Comment:
No need to make this public. I can't think of many cases that programs would mix owned and non-owned Pids and require distinguishing them. Making at an implementation detail also allows changing the underlying implementation in the future, if necessary.